tonypedraza/alloy
# Add drf

Added DRF to the project.

To test:
Checkout the PR and restart the docker containers. Visit localhost:8000/backend and you should see the DRF api.
